examples/speech_to_text_v1.py

```diff
@@ -17,4 +18,4 @@
     print(json.dumps(speech_to_text.recognize(
         audio_file, content_type='audio/wav', timestamps=True,
         word_confidence=True),
-        indent=2))
+                     indent=2))
```

Comment:
It doesn't change the meaning, but I suspect this extra indentation wasn't intended.

Reply:
This was a pylint complain, so added those indentation